Sources/OpenAPIRuntime/URICoder/Serialization/URISerializer.swift

```diff
@@ -50,16 +50,16 @@ extension CharacterSet {
 
     /// A character set of unreserved symbols only from RFC 6570 (excludes
     /// alphanumeric characters).
-    fileprivate static let unreservedSymbols: CharacterSet = .init(charactersIn: "-._~")
+    fileprivate nonisolated(unsafe) static let unreservedSymbols: CharacterSet = .init(charactersIn: "-._~")
```

Comment:
This would be better fixed by Foundation.CharacterSet being Sendable on Linux (it already is on Darwin).

Reply:
For reference: https://github.com/apple/swift-openapi-runtime/pull/107#issuecomment-2180542772